go/cmd/dolt/commands/sqlserver/sqlserver.go

```diff
@@ -541,3 +548,127 @@ func getYAMLServerConfig(fs filesys.Filesys, path string) (ServerConfig, error)
 
 	return cfg, nil
 }
+
+// BuildConnectionStringQueryist returns a Queryist that connects to the server specified by the given server config. Presence in this
+// module isn't ideal, but it's the only way to get the server config into the queryist.
+func BuildConnectionStringQueryist(ctx context.Context, cwdFS filesys.Filesys, creds *cli.UserPassword, apr *argparser.ArgParseResults, host string, port int, useTLS bool, dbRev string) (cli.LateBindQueryist, error) {
```

Comment:
Move this into it's own file, but keep it in the same package. It clutters up this one - probably shouldn't have been in the sqlclient.go file to begin with.